{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2020-24119",
      "description": "A heap buffer overflow read was discovered in upx 4.0.0, because the check in p_lx_elf.cpp is not perfect.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-24119",
      "reportURL": "https://github.com/upx/upx/issues/388",
      "timeStamp": "2021-05-28T19:43:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2020-24119",
      "url": "https://github.com/Live-Hack-CVE/CVE-2020-24119",
      "added_at": "2022-02-19T16:34:00Z"
    },
    {
      "id": "CVE-2020-24119",
      "url": "https://github.com/upx/upx/issues/388",
      "added_at": "2022-02-19T16:34:00Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2020-24119",
      "url": "https://github.com/Live-Hack-CVE/CVE-2020-24119",
      "added_at": "2022-02-19T16:34:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2020-24119",
      "url": "https://github.com/upx/upx/issues/388",
      "added_at": "2021-05-28T19:43:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
